blocklist-client/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "blocklist-client"
+version = "0.0.1"
+edition = "2021"
+
+[[bin]]
+name = "blocklist-client"
+path = "src/main.rs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+warp = "0.3"
+tokio = { version = "1", features = ["full"] }
```

Comment:
Nit: We might be able to get away with using fewer tokio features. I think we need only `rt-multi-thread` feature at this time.

Reply:
sure, I can narrow it down. It requires: `rt-multi-thread`, `rt`, `macros`